api/vulnerability_exceptions_container.go

```diff
@@ -0,0 +1,85 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+func (svc *VulnerabilityExceptionsService) CreateVulnerabilityExceptionsContainer(vuln VulnerabilityException) (
+	response VulnerabilityExceptionContainerResponse, err error) {
+	err = svc.client.RequestEncoderDecoder("POST", apiV2VulnerabilityExceptions, vuln, &response)
+	return
+}
+
+func (svc *VulnerabilityExceptionsService) GetVulnerabilityExceptionsContainer(guid string) (response VulnerabilityExceptionContainerResponse, err error) {
+	if guid == "" {
+		err = errors.New("specify a Guid")
+		return
+	}
+	apiPath := fmt.Sprintf(apiV2VulnerabilityExceptionFromGUID, guid)
+	err = svc.client.RequestDecoder("GET", apiPath, nil, &response)
+	return
+}
+
+func (svc *VulnerabilityExceptionsService) ListVulnerabilityExceptionsContainers() (response VulnerabilityExceptionContainerResponse, err error) {
+	err = svc.client.RequestDecoder("GET", apiV2VulnerabilityExceptions, nil, &response)
+	return
+}
+
+func (svc *VulnerabilityExceptionsService) UpdateVulnerabilityExceptionsContainer(data VulnerabilityException, id string) (
```

Comment:
Why do we ask for the `id` if the`VulnerabilityException` struct already has the `Guid` field? feels redundant

Reply:
Unfortunately, the request body is invalid if the guid field is present. I believe there is another resource we had to do something similar.

Reply:
@dmurray-lacework I think that is how we have done it in LQL, TeamMembers, and other v2 schemas. We should be consistent and follow the adopted pattern. (which is set the id and strip it in the update func)